pyspark-interpreter/src/main/scala/org/apache/toree/kernel/interpreter/pyspark/PySparkInterpreter.scala

```diff
@@ -148,4 +154,19 @@ class PySparkInterpreter(
   // Unsupported
   override def doQuietly[T](body: => T): T = ???
 
+  override def languageInfo: LanguageInfo = {
+    if ((!pySparkService.isRunning) || (!pySparkState.isReady)) {
+      LanguageInfo(
+        "python",
+        version = "UNKNOWN",
```

Comment:
I guess we can subprocess call python before we initialize it to get a --version?